pennylane_lightning/core/src/simulators/lightning_gpu/CMakeLists.txt

```diff
@@ -65,6 +67,15 @@ set_property(TARGET ${PL_BACKEND} PROPERTY POSITION_INDEPENDENT_CODE ON)
 # string(REPLACE "libcudart_static.a" "libcudart.so" CUDA_SHARED_RT "${CUDA_LIBRARIES}")
 set_target_properties(${PL_BACKEND} PROPERTIES INSTALL_RPATH "$ORIGIN/../cuquantum/lib:$ORIGIN/../cuquantum/lib64:$ORIGIN/")
 
+if(ENABLE_MPI)
+    if(NOT MPI_LIBRARY)
+        message(FATAL_ERROR "MPI_LIBRARY not found. Please set MPI_LIBRARY to the path of the MPI libraries.")
```

Comment:
Could it also work with the MPI dir path, i.e., `MPI_LIB_DIR` ?

Reply:
MPI_LIB_DIR does work on isaic